cmd/create/machinepool/nodepool.go

```diff
@@ -72,7 +72,7 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		clusterVersion := cluster.Version().RawID()
 		// This is called in HyperShift, but we don't want to exclude version which are HCP disabled for node pools
 		// so we pass the relative parameter as false
-		versionList, err := versions.GetVersionList(r, channelGroup, true, true, false, false)
+		_, versionList, err := versions.GetVersionList(r, channelGroup, true, true, false, false)
```

Comment:
Should we also use the defaultVersion [here](https://github.com/openshift/rosa/blob/b6ca8f1112885cfb56cab8f4ba3a1a734005c195/cmd/create/machinepool/nodepool.go#L103)?

Reply:
for node pool default version is the cluster version (aka control plane). So no need to change it.